<?xml version="1.0"?>
<robot name="pbr_mesh">
  <link name="base_link">
    <inertial>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <mass value="0.105"/> <!-- Mass in kilograms -->
      <inertia ixx="3.08265843e-07" ixy="-1.52428994e-08" ixz="-9.90446860e-08"
               iyy="3.70931582e-07" iyz="-3.17292000e-08"
               izz="1.58639665e-07"/> <!-- Moments of inertia in kg*m^2 -->
    </inertial>
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <geometry>
        <mesh filename="file:///home/akshay/mesh_files/sacled_2.obj" scale="1.0 1.0 1.0"/>
      </geometry>
      <material name="grey">
        <color rgba="0.5 0.5 0.5 1"/>
      </material>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <geometry>
        <mesh filename="file:///home/akshay/mesh_files/sacled_2.obj" scale="1.0 1.0 1.0"/>
      </geometry>
    </collision>
  </link>
</robot>
